gulpfile.js

```diff
@@ -82,6 +83,66 @@ function createReloadServer() {
   });
 }
 
+function writeServiceWorkerFile() {
+  let path = require('path');
+  let rootDir = 'dist';
+
+  let config = {
+    cacheId: 'polymerproject',
+    staticFileGlobs: [
+      `${rootDir}/1.0/index.html`,
+      `${rootDir}/1.0/shell`,
```

Comment:
Right, the code snippet I provided at https://github.com/Polymer/docs/pull/1641#issuecomment-241100911 removes a few unneeded files from `staticFileGlobs`.